examples/bert/yamls/test/main.yaml

```diff
@@ -67,16 +65,16 @@ optimizer:
   eps: 1.0e-08
   weight_decay: 0.0
 
-grad_clip_norm: -1.0
-
```

Comment:
is it worthwhile to add the `gradient_clipping` algorithm or is that not needed for this test?

Reply:
AFAIK, we've never used it with BERT. I removed this line because it is no longer referenced in the script.